website/docs/docs/dbt-cloud/dbt-cloud-changelog.md

```diff
@@ -4,6 +4,28 @@ id: "cloud-changelog"
 sidebar_label: Changelog
 description: "Changelog for the dbt Cloud application"
 ---
+## dbt Cloud v1.1.28 (June 9, 2021)
+We shipped a far better experience for GitLab users. Be sure to check out new CI features that are now available for customers using GitLab, and all developers should test out Slim CI which will speed up their model builds.
+
+#### New products and features
+
+- `Slim CI`: We’ve made Slim CI available for all our cloud customers! With Slim CI, you don't have to rebuild and test all your models; you can instruct dbt Cloud to run jobs on only modified or new resources. If you are a GitHub or GitLab user, try creating a new job that runs on pull requests and you can signal to dbt to run only on these modified resources by including the `state:modified` argument. Read more about Slim CI [here](https://docs.getdbt.com/docs/dbt-cloud/using-dbt-cloud/cloud-enabling-continuous-integration-with-github#slim-ci).
```

Comment:
I might, for consistency, say `state:modified+` for the argument since you definitely want to rebuild models downstream from the change.

Reply:
Okay - I considered that, but felt the + was a separate flag to get all the children models too. I'll change it though to avoid any confusion. Thanks for reading!

Reply:
I realized the original article wasn't clear about this, either, so submitted a PR there too since in a SlimCI context it's *super* important to include children lest a change to a parent cause a problem that would break a child model.